### DISCUSSION TITLE
DietPi-Software | Seafile

### DISCUSSION THREAD

**Discussion by msongz** [MichaIng/DietPi#5676] [Software Request]
#### Give us some formal software information:
- Software title | seafile
- Short description | File syncing and sharing software with file encryption and group sharing, emphasis on reliability and high performance.
- GitHub URL (if available) | https://github.com/haiwen/seafile
- External URL (if available) | http://seafile.com/

**Comment by MichaIng** [comment]
Added to FeatHub, feel free to vote for it: https://feathub.com/MichaIng/DietPi/+10
